config-templates/tables.sql

```diff
@@ -1,38 +1,55 @@
---Statistics for IdPs
-CREATE TABLE statistics (
-    year INT NOT NULL,
-    month INT NOT NULL,
-    day INT NOT NULL,
-    sourceIdp VARCHAR(255) NOT NULL,
-    service VARCHAR(255) NOT NULL,
-    count INT,
-    INDEX (sourceIdp),
-    INDEX (service),
-    PRIMARY KEY (year, month, day, sourceIdp, service)
+# daily and monthly logins and unique users for all combinations of idp+sp
```

Comment:
comments in SQL should start with double dash: --

Reply:
This is [MySQL](https://dev.mysql.com/doc/refman/8.0/en/comments.html), the commands would not work generally, but I can change the comment syntax to general SQL

Reply:
add9123